ui/components/ui/list-item/index.scss

```
@@ -23,14 +26,17 @@
   &:focus-within {
     background-color: var(--Grey-000);
   }
-
+  &__topWrap{
```

```suggestion
  &__top-wrap{
```
- To be consistent with the coding style

---

fixed in a78ec32a4291acbd25e7479c9891d86dfb1b8c82